File: src/androidTest/java/pt/joaomneto/titancompanion/TestHOTW.java
package pt.joaomneto.titancompanion;


import android.support.test.filters.LargeTest;
import android.support.test.runner.AndroidJUnit4;

import org.junit.runner.RunWith;

import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook;


@LargeTest
@RunWith(AndroidJUnit4.class)
public class TestHOTW extends TestST {

    @Override
    protected FightingFantasyGamebook getGamebook() {
        return FightingFantasyGamebook.HOWL_OF_THE_WEREWOLF;
    }


}

File: src/androidTest/java/pt/joaomneto/titancompanion/TestHOTW.kt
package pt.joaomneto.titancompanion

import androidx.test.ext.junit.runners.AndroidJUnit4
import androidx.test.filters.LargeTest
import org.junit.runner.RunWith
import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook

@LargeTest
@RunWith(AndroidJUnit4::class)
class TestHOTW : TestST() {

    override val gamebook = FightingFantasyGamebook.HOWL_OF_THE_WEREWOLF
}
